### ISSUE TITLE
State Graph not drawing properly when creating a new state machine, flooding the logs with an error.

### ISSUE THREAD

**Issue by DraconicMentalist** [imjp94/gd-YAFSM#73]
I am operating from Godot 1.4.2 rc1, but have found the bug to be present in 4.1.1 stable as well. Attempting to either open or create a new state machine results in this error being thrown repeatedly:

`servers/rendering/renderer_canvas_cull.cpp:1084 - Condition "p_colors.size() != 1 && p_colors.size() * 2 != p_points.size()" is true.`

The editor does not render the state graph correctly, displaying any state nodes without a grid background. Let me know if any further information is needed.

**Comment by rsubtil** [comment]
Comparing the documentation for `draw_multiline_colors` between [3.6](https://docs.godotengine.org/en/3.6/classes/class_canvasitem.html#class-canvasitem-method-draw-multiline-colors) and [4.0](https://docs.godotengine.org/en/stable/classes/class_canvasitem.html#class-canvasitem-method-draw-multiline-colors), it seems that before you had to specify a color for each point, but now you need to specify a color for each 2 points.

Removing these lines fixed in my case. I never used this drawing functionality so I'm not sure this breaks anything:
```diff
diff --git a/addons/imjp94.yafsm/scenes/flowchart/FlowChartGrid.gd b/addons/imjp94.yafsm/scenes/flowchart/FlowChartGrid.gd
index 70c9581..94363ec 100644
--- a/addons/imjp94.yafsm/scenes/flowchart/FlowChartGrid.gd
+++ b/addons/imjp94.yafsm/scenes/flowchart/FlowChartGrid.gd
@@ -43,7 +43,6 @@ func _draw():
         var base_ofs = i * snap
 
         multi_line_vector_array.append(Vector2(base_ofs, offset.y))
-        multi_line_color_array.append(color)
         multi_line_vector_array.append(Vector2(base_ofs, corrected_size.y))
         multi_line_color_array.append(color)
 
@@ -59,7 +58,6 @@ func _draw():
         var base_ofs = i * snap
 
         multi_line_vector_array.append(Vector2(offset.x, base_ofs))
-        multi_line_color_array.append(color)
         multi_line_vector_array.append(Vector2(corrected_size.x, base_ofs))
         multi_line_color_array.append(color)
 
```

**Comment by imjp94** [comment]
This has been fixed in #65 , just pull the latest update from master branch
